### ISSUE TITLE
Refactor buttons with Aphrodite

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#7168]
**Describe the issue you encountered:** `/js/components/button.js` is called so many places that refactoring it will help other stuff to be refactored as well :-)

- Any related issues: #6783

**Comment by luixxiul** [comment]
I added priority/high label (as the refactor is needed by #6785) with release/not-blocking.

**Comment by luixxiul** [comment]
This ticket can be closed as well as #9071 after `navItem` is added, though I am not quite sure which button corresponds to that. 

https://github.com/luixxiul/browser-laptop/blob/96c6445d04a8ddccaf0bbb6fac6bfa0a00d9c428/app/renderer/components/common/browserButton.js#L29
````
      // this.props.navItem && styles.browserButton_navItem,
````

@cezaraugusto do you have an idea?
